src/main/java/org/sert2521/gamename/Drivetrain/Drivetrain.kt

```diff
@@ -0,0 +1,51 @@
+package org.sert2521.gamename.Drivetrain
+
+import edu.wpi.first.wpilibj.command.Command
+import edu.wpi.first.wpilibj.command.Subsystem
+import org.sert2521.gamename.FRONT_LEFT_MOTOR
+import org.sert2521.gamename.FRONT_RIGHT_MOTOR
+import org.sert2521.gamename.OI
+import org.sert2521.gamename.REAR_LEFT_MOTOR
+import org.sert2521.gamename.REAR_RIGHT_MOTOR
+import org.strongback.components.Motor
+import org.strongback.drive.TankDrive
+import org.strongback.hardware.Hardware
+
+/**
+ * This is the Drivetrain subsystem, which is in charge of controlling the speeds of the individual
+ * talons and converting joystick input to movement speeds.
+ */
+object Drivetrain : Subsystem() {
```

Comment:
Why did we move this? Especially don't approve of capitals in a package name, that's a big no-no. Also, for the base I'd rather have everything in the root package so we can deal with organization as we add more stuff in the real codebase.